test/simple_optimizers.jl

```diff
@@ -3,13 +3,14 @@
 N = 1000
 dim_x = 10
 dim_y = 8
-σ = 0.01
+σ = 0.0
 epochs = 200
 
 ## True model
 
 true_model = Dense(dim_x, dim_y; bias=false)
-true_optimizer = Dict("argmax" => one_hot_argmax, "ranking" => ranking)
+true_optimizer = Dict("argmax" => argmax_optimizer, "ranking" => ranking)
+cost(y; instance) = -true_model(instance)' * y
```

Comment:
Would it be more natural in terms of code to maximize the negative cost and call it benefit or something?

Reply:
If we use `PerturbedCost` as a loss, its `cost` argument is minimized, so we have to use this formulation here I think